chanacceptor/rpcacceptor.go

```diff
@@ -356,6 +356,30 @@ func (r *RPCAcceptor) sendAcceptRequests(errChan chan error,
 				):
 					commitmentType = lnrpc.CommitmentType_SIMPLE_TAPROOT
 
+				case channelFeatures.OnlyContains(
+					lnwire.SimpleTaprootOverlayChansRequired,
```

Comment:
so taproot channels don't need the static_remote_key option because they support the static remote key as a standard from the beginning ?

Reply:
Correct, all channel types starting from anchor channels basically have that feature built in.